src/Ztm.Zcoin.NBitcoin/UInt256JsonConverter.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using NBitcoin;
+using Newtonsoft.Json;
+
+namespace Ztm.Zcoin.NBitcoin
+{
+    public sealed class UInt256JsonConverter : JsonConverter<uint256>
```

Comment:
We should move this one to a separated project (e.g. `Ztm.Zcoin.NBitcoin.Json`) due to in required additional dependencies.